crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -270,6 +270,8 @@ where
             .validator
             .validate_version_specific_fields(EngineApiMessageVersion::V4, payload_or_attrs)?;
 
+        validate_execution_requests(&execution_requests)?;
```

Comment:
this seems okay here because these rules are for the request object specifically, unclear how this will look like for OP V4